lib/components/button-group/button-group.less

```diff
@@ -58,7 +58,6 @@
         & .s-btn--text {
             &:before {
                 content: attr(data-text);
-                content: attr(data-text) / "";
```

Comment:
I realized that we have a duplicate property here, so I got rid of it in https://github.com/StackExchange/Stacks/pull/1894/commits/56a606c9251371cc4fc1a9bbecd308d79a712442